lib/ImageResizer.js

```diff
@@ -33,11 +34,17 @@ ImageResizer.prototype.resizeRoute = function({originalRouteName, resizedRouteNa
     }
     return Buffer.concat(buffers);
   }).then(buffer => {
-    let resizer = sharp(buffer).resize(width, height);
+    let image = sharp(buffer);
+    if (!disableRotation)
+    {
+      image = image.rotate();
+    }
+    let noEnlargement = false;
     if (!allowEnlargement) {
-      resizer.withoutEnlargement();
+      noEnlargement = true;
     }
-    const resizedPromise = resizer.toBuffer();
+    image = image.resize(width, height, { withoutEnlargement: noEnlargement });
```

Comment:
This is equivalent to:
```suggestion
    image = image.resize(width, height, { withoutEnlargement: !allowEnlargement });
```